x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface EnterpriseSearchEnginesResponse {
+  meta: {
+    from: number;
+    size: number;
+    total: number;
+  };
+  results: Array<{
+    name: string;
+    indices: string[];
```

Comment:
LGTM 💯 
 
Just wanted to confirm, we would be waiting for `ent-search` API to support `last_updated` and `document_count` fields before supporting in the FE?

Reply:
correct, we can default the `document_count` to 0 for now. and we can discuss the updates needed to `ent-search` in the sync tomorrow.